### PR TITLE
Improve the documentation.

### PR DESCRIPTION
**Pull request by jcvernaleo**

### CODE REVIEW

**Comment by jcvernaleo** [comment]
And since @jrick suggested it, here is the pretty version:
https://github.com/jcvernaleo/decred-release/blob/2a1f4f0b7b2d1311ba661fc49749ed8f8413448b/cmd/dcrinstall/README.md

**Comment by jolan** [comment]
Would be nice to include the git/glide/go steps to build from source like we do for other repos.

File: cmd/dcrinstall/README.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+# dcrinstall
+
+dcrinstall is a tool to automate the install and upgrade process for
+the decred software.
```
Comment (jrick) [inline cmd/dcrinstall/README.md]: Maybe also mention that it automates the setup

File: cmd/dcrinstall/README.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+~/decred on a UNIX type OS).
+
+The dcrinstall tool records all actions in %HOMEPATH%\decred\dcrinstall.log
+(or ~/decred/dcrinstall.log on a UNIX type OS).
+
+### Windows
+
+Configuration files:
+```
+%HOMEPATH%\AppDate\Local\Dcrctl\dcrctl.conf
```
Comment (jrick) [inline cmd/dcrinstall/README.md]: `%LOCALAPPDATA%\Dcrctl\dcrctl.conf`

**Comment by jrick** [comment]
@jolan I believe dcrinstall is meant to be distributed as a binary, and a typical user won't need to build it from source (so they won't care about glide).

**Comment by jolan** [comment]
@jrick Right, this would be for atypical users who like to build themselves, security conscious people who dislike binary distributions, or developers who still want the steps.

**Comment by jrick** [comment]
Well even if they are added, i believe the build instructions should be kept separate from the usage instructions.  The usage instructions also assume that the binary was downloaded from a web browser since it runs it out of the Downloads folder, so combining the instructions would make even less sense.

**Comment by jcvernaleo** [comment]
So after talking to @marcopeereboom I added in the build instrutions but tried to call them out as not necessary for most users.

**Comment by jcvernaleo** [comment]
So this is the latest rendered version: https://github.com/jcvernaleo/decred-release/blob/741c36b324ed6f82b4678b92f11ae45939b61fcb/cmd/dcrinstall/README.md

File: cmd/dcrinstall/README.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+In install mode dcrinstall downloads the latest released binaries of
+dcrd, dcrwallet, dcrctl, and dcrticketbuyer for your operating system
+and platform, installs them, sets up the config files, and creates a
+wallet for you.  In upgrade mode, dcrinstall replaces your binaries
+with the latest copies but makes no changes to your configs.
+
+## Upgrading for an existing installation
+
+The following steps are required to upgrade a system with decred that
+was not installed by dcrinsall.  If you already have decred installed
```
Comment (jolan) [inline cmd/dcrinstall/README.md]: typo s/dcrinsall/dcrinstall/

**Comment by jcvernaleo** [comment]
Thanks @jolan, fixed that.

File: cmd/dcrinstall/README.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+will ONLY work on a fresh install (as in you have no existing
+configuration files on the computer).  Just add the option -net
+Testnet to the command:
+
+
+```
+dcrinstall.exe -net Testnet
+```
+
+or on OXS/UNIX:
```
Comment (jolan) [inline cmd/dcrinstall/README.md]: s/OXS/OSX/

File: cmd/dcrinstall/README.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+dcrinstall is a tool to automate the install, upgrade, and setup
+process for the decred software.
+
+In install mode dcrinstall downloads the latest released binaries of
+dcrd, dcrwallet, dcrctl, and dcrticketbuyer for your operating system
+and platform, installs them, sets up the config files, and creates a
+wallet for you.  In upgrade mode, dcrinstall replaces your binaries
+with the latest copies but makes no changes to your configs.
+
+## Upgrading for an existing installation
```
Comment (jolan) [inline cmd/dcrinstall/README.md]: I think "Upgrading an existing installation" sounds better.

**Comment by jcvernaleo** [comment]
@jolan agreed, fixed that and the OSX thing (plus something marco noticed).

**Comment by jolan** [comment]
Just went through it again and didn't see anything.

OK

**Comment by alexlyp** [comment]
OK

**Pull request merged**